airflow/cli/commands/task_command.py

```diff
@@ -46,7 +46,7 @@
 from airflow.models.operator import needs_expansion
 from airflow.models.param import ParamsDict
 from airflow.models.taskinstance import TaskReturnCode
-from airflow.settings import IS_K8S_EXECUTOR_POD
+from airflow.settings import IS_EXECUTOR_CONTAINER, IS_K8S_EXECUTOR_POD
```

Comment:
New mechanism to enable task log output to show up in task logs for containerized executors. A previous mechanism existed for K8s only (a source of executor coupling!) a new one was added, but K8s was not migrated as part of this PR to keep the size down. More changes in settings.py and logging_mixin.py below